src/app/layout.tsx

```diff
@@ -33,8 +33,11 @@ export default async function RootLayout({
         <meta charSet="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
       </head>
-      <body className={ibm.className}>
+      <body className={`${ibm.className} text-primaryT-darker`}>
         <Providers>
+          <nav>
```

Comment:
Do you really have to put inside another <nav> tag?

Reply:
yes, it's semantic tag for SEO performance

Reply:
actually nah, nextui component handles this